include/shell/shell_fprintf.h

```diff
@@ -34,6 +34,11 @@ struct shell_fprintf {
 	struct shell_fprintf_control_block *ctrl_blk;
 };
 
+enum shell_flag {
+	SHELL_FLAG_NLCR_DEFAULT,	/* Do not map NL or CR */
```

Comment:
I think the standard names for those are `LF` and `CR`. Can we use those instead?